asciidoc-confluence-publisher-maven-plugin/src/main/java/org/sahli/asciidoc/confluence/publisher/maven/plugin/AsciidocConfluencePublisherMojo.java

```diff
@@ -47,61 +47,63 @@
 @Mojo(name = "publish")
 public class AsciidocConfluencePublisherMojo extends AbstractMojo {
 
+    static final String PREFIX = "asciidoctor.confluence.";
```

Comment:
I am not absolutely sure about using a prefix, and if so, which one (e.g. something including `confluence` and `publisher` are a fqn might be better). Did you find any best practice on prefixes for plugin-specific command line options?

Reply:
I did not looked on what are the best practices.  I used this options because it's the same way the org.asciidoctor:asciidoctor-maven-plugin uses.